src/SignalR/clients/ts/signalr/src/LongPollingTransport.ts

```diff
@@ -106,7 +108,7 @@ export class LongPollingTransport implements ITransport {
 
     private updateHeaderToken(request: HttpRequest, token: string | null) {
         if (!request.headers) {
-            request.headers = {};
+            request.headers = this.headers;
```

Comment:
revert this

Reply:
Fixed: 886181de8c6b3fd00ddbcc1fc79f52d4c4c077ed